ARC specification.md

```diff
@@ -296,102 +302,156 @@ Notes:
+
+Format and scope of these cases naturally vary depending on the type of ARC, aim of the containing validation package and tools used for creating and performing the validation. 
+Therefore, no further requirements are made on the format of validation cases.
+
+  example:
+
+  > The following example shows a validation case simply defined using natural language.
+
+  ```
+  All Sample names in this ARC must be prefixed with the string "Sample_"
+  ```
+
+  Any ARC where all sample names are prefixed with the string "Sample_" would be considered valid in regard to this validation case.
+
+### Validation packages
+
+A **validation package** bundles a collection of [validation cases](#validation-cases) that the ARC MUST pass to qualify as _valid_ in regard to the _validation package_ with instructions on how to perform the validation and summarize the results.
+
+Validation packages
+
+- MUST be executable. 
+  This can for example be achieved by implementing them in a programming language, a shell script, or a workflow language.
+
+- MUST validate an ARC against all contained validation cases upon execution.
+
+- MUST have a globally unique name.
+  This will eventually be enforced by a central validation package registry
+
+- SHOULD be versioned using [semantic versioning](https://semver.org/)
+
+- MUST create a `validation_report.*` file upon execution that summarizes the results of validating the ARC against the cases defined in the validation package.
+  The format of this file SHOULD be of an established test result format such as [JUnit XML](https://github.com/windyroad/JUnit-Schema) or [TAP](https://testanything.org/).
+
+- MUST create a `badge.svg` file upon execution that visually summarizes the results of validating the ARC against the validation cases defined in the validation package.
+  The information displayed SHOULD be derivable from the `validation_report.*` file and MUST include the _name_ of the validation package.
+
+### Reference implementation
+
+A reference implementation for creating validation cases, validation packages, and validating ARCs against them is provided in the [arc-validate software suit](https://github.com/nfdi4plants/arc-validate)
+
+## Continuous quality control
+
+In addition to manually validate ARCs against validation packages, ARCs MAY be continuously validated against validation packages using a continuous integration (CI) system. 
+This process is further referred to as _Continuous Quality Control_ (CQC) of the ARC. CQC can be triggered by any event that is supported by the CI system, e.g. a push to a branch of the ARC repository or a pull request.
+
+### The cqc branch
+
+To make sure that validation results are bundled with the ARC but do not pollute their commit history, validation results MUST be stored in a separate branch of the ARC repository.
```

Comment:
pollute `its`  commit history

Reply:
i think ill go for plural in both cases, 
> To make sure that validation results are bundled with ARCs but do not pollute their commit history